Formula/spot.rb

```diff
@@ -0,0 +1,15 @@
+class Spot < Formula
+  desc ": a platform for LTL and ω-automata manipulation"
```

Comment:
```suggestion
  desc "Platform for LTL and ω-automata manipulation"
```

Reply:
I changed the description and added a livecheck as you suggested. Thank you for your feedback.